cmd/tiller-proxy/internal/handler/handler_test.go

```diff
@@ -378,6 +378,58 @@ func TestActions(t *testing.T) {
 			},
 			ResponseBody: `{"data":[{"releaseName":"foobar","version":"","namespace":"default","status":"DEPLOYED","chart":"","chartMetadata":{}}]}`,
 		},
+		{
+			// Scenario params
+			Description: "Rolls back a release",
+			ExistingReleases: []release.Release{
+				release.Release{Name: "foo", Namespace: "default", Info: &release.Info{Status: &release.Status{Code: release.Status_DEPLOYED}}},
+			},
+			DisableAuth:      false,
+			ForbiddenActions: []auth.Action{},
+			// Request params
+			RequestBody: `{"chartName": "foo", "releaseName": "foo",	"version": "1.0.0"}`,
+			RequestQuery: "?version=1",
+			Action:       "rollback",
+			Params:       map[string]string{"namespace": "default", "releaseName": "foo"},
+			// Expected result
+			StatusCode: 200,
+			RemainingReleases: []release.Release{
+				release.Release{Name: "foo", Namespace: "default", Info: &release.Info{Status: &release.Status{Code: release.Status_DEPLOYED}}},
+			},
+			ResponseBody: `{"data":{"name":"foo","info":{"status":{"code":1}},"namespace":"default"}}`,
+		},
+		{
+			// Scenario params
+			Description:      "Rollsback a missing release",
+			ExistingReleases: []release.Release{},
+			DisableAuth:      true,
+			ForbiddenActions: []auth.Action{},
+			// Request params
+			RequestBody: `{"chartName": "foo", "releaseName": "foobar",	"version": "1.0.0"}`,
+			RequestQuery: "?version=1",
```

Comment:
I see both `version` and `revision` used for the helm release version of an installed chart, though the [helm docs seem to use `revision` consistently (or try to)](https://github.com/helm/helm/blob/master/docs/helm/helm_rollback.md) the `helm` api below uses `RollbackVersion`. Hmph. Should we just use "revision" if that's what it is going forward (and to avoid confusion when chart version is used in the same context, as here in the previous line)?

Reply:
that's a good point, I was using `version` because that's how they call it in the API but I guess that's because historical reasons. I am okay if we use `revision` because there are a lot of versions already (`appVersion`, `chartVersion`...)